app/views/users/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="container text-center">
+<h2> <%= t(:hello)%> Users </h2>
+  <div class="row justify-content-md-center">
+    <%= link_to "#{t '.add_user' }", new_user_path(locale: I18n.locale) %>
```

Comment:
As u are passing the locale in the URL but the problem is, you have to pass & extract a locale in each URL every time.
But this is not a proper solution, one of the solutions is, either you store the locale in a cookie and set the locale before each request in application controller.